cli/lib/kontena/cli/plugins/install_command.rb

```diff
@@ -0,0 +1,27 @@
+require 'open3'
+
+module Kontena::Cli::Plugins
+  class InstallCommand < Clamp::Command
+
+    parameter 'NAME', 'Plugin name'
+
+    def execute
+      install_plugin(name)
+    end
+
+    def install_plugin(name)
+      plugin = "kontena-plugin-#{name}"
+      gem_bin = `which gem`.strip
```

Comment:
You can replace `which` by traversing through $PATH and picking up the first hit. That's what "which" does. This should work:

```ruby
def which(cmd_name)
  ENV["PATH"].split(File::PATH_SEPARATOR).each do |path|
    Dir[File.join(File.expand_path(path), cmd_name)].each do |file_path|
      return file_path if File.executable?(file_path) && !File.directory?(file_path)
    end
  end
end

```